webrtc.html

```diff
@@ -8800,6 +8806,9 @@ <h3>
               [=RTCRtpSender/list of implemented send codecs=], with the "enabled" flag
               set in an implementation defined manner.
             </p>
+            <p>
+              Let <var>sender</var> have a <dfn data-dfn-for="RTCRtpSender">[[\NegotiatedCodecs]]</dfn> internal slot, consisting of {{RTCRtpCodecParameters}}, and initialized to an empty list.
```

Comment:
Ah, this says [[NegotiatedCodecs]] contains RTCRtpCodecParameters, not tuples. Good, but means the code above needs to be fixed to not add tuples.

Reply:
Or rather ... construct an RTCRtpCodecCapability from the RTCRtpCodec and add the PT from the description.